CMU-313/spring23-nodebb-green
# Fully-integrate career feature

## Description
* Merge in changes from draft PR of career-model
* Created new route that calls the controllers.write method `Career.register`
* Updated career-model to Flask app so that it can be called at an endpoint
* Added deployment configuration so that endpoint can be called from https://nodebb-green-career.fly.dev/predict

## Motivation and Context
Fully integrate career feature to allow for students to submit their information to recruiters, and let recruiters see a prediction on how well the student will do in the field.

## How Has This Been Tested?
1. At https://nodebb-green.fly.dev/, create a new student account (or sign into existing student account
2. Click the new career tab:
    ![image](https://user-images.githubusercontent.com/50491000/230795888-fde00d2e-be96-4e59-b889-595ba9eb7b7b.png)
3. Fill in student information with "high" stats (high GPA, many programming languages, and many internships)
4. Repeat steps 1-3, but with a student that has "low" stats (low GPA, only 1 programming language, few internships)
5. Create a new account, selecting "Recruiter" for account type
6. In the career tab, verify that both students are viewable, and the student with "high" stats has a 1 prediction, and the student with "low" stats has a 0 prediction

## Types of changes
